examples/boids/Cargo.toml

```diff
@@ -12,4 +12,4 @@ getrandom = { version = "0.2", features = ["js"] }
 rand = "0.8"
 serde = { version = "1.0", features = ["derive"] }
 yew = { path = "../../packages/yew" }
-gloo = { git = "https://github.com/rustwasm/gloo/" }
+gloo = "0.3.0"
```

Comment:
We should update this in a separate PR. Yew uses `0.2` so it would cause gloo to be added twice. Also, the `log` crate should be switched out for `gloo-console` so the gloo update can be done in their PR

Reply:
Reverted back